Updates dribbling device text by removing unnecessary text and adding an explicit note about the required visible percentage of ball area (#32)
robocup-ssl-br/rules-ssl-el: chapters/robots.adoc

==== Dribbling Device
The dribbling device present in the Regular League will not be allowed in the SSL-EL. Robots can have the device, but it will not be allowed to be activated during the game.
 

> The definition fo a so-called "dribbling device" is: a device that actively exert spin on the ball, which keeps the ball in contact with the robot.

Reviewer: ```suggestion
The definition of a so-called "dribbling device" is as follows: a device that actively exerts spin on the ball and keeps the ball in contact with the robot.
```
